llvm_ir_dataset_utils/tools/RemoveFunctionBodyPass/CMakeLists.txt

```diff
@@ -0,0 +1,36 @@
+cmake_minimum_required(VERSION 3.20)
+
+project(RemoveFunctionBodyPass)
+
+set(LT_LLVM_INSTALL_DIR "" CACHE PATH "LLVM installation directory")
```

Comment:
Why do we need to set this path? Doesn't `opt` support explicitly passing the shard object path?